Example/Custom Integration/MyAPIClient.h

```diff
@@ -33,10 +33,11 @@ typedef void (^STPCreateSetupIntentCompletionHandler)(MyAPIClientResult status,
  method signature is the most interesting part: you need some way to ask *your* backend to create
  a PaymentIntent with the correct properties, and then it needs to pass the client secret back.
  
+ @param additionalParameters additional parameters to pass to the example backend
  @param completion completion block called with status of backend call & the client secret if successful.
  @see https://stripe.com/docs/payments/payment-intents/ios
  */
-- (void)createPaymentIntentWithCompletion:(STPPaymentIntentCreationHandler)completion;
+- (void)createPaymentIntentWithCompletion:(STPPaymentIntentCreationHandler)completion additionalParameters:(NSString * _Nullable)additionalParameters;
```

Comment:
Is this going to convert to swift strangely? I'd probalby expect `createPaymentIntentWithAdditionalParameters:(NSString * _Nullable)additionalParameters completion:(STPPaymentIntentCreationHandler)completion;`

I'd also probably expect `additionalParameters` to be an NSDictionary but since this is sample backend code don't feel strongly on either of these

Reply:
Yeah, that was roughly my feeling. It should take a dictionary and convert all the parameters properly, but this was a quick change to make an Objective-C-only sample app work, so meh. I'm more worried about how this additional parameter adds incremental complexity to our sample app.